BenchTesting.java

```diff
@@ -62,5 +65,87 @@ public static void main(String[] args) {
         System.out.println("Expected: temp1. Actual: " + island1.getPassDetails(123));
         System.out.println("Expected: true. Actual: " + island1.isPassOnIsland(pass1));
         System.out.println("Expected: false. Actual: " + island1.isPassOnIsland(pass2));
+//Testing for Ferry class-> To test everything for this class, you need to comment the Island and Pass variables above ^
+
```

Comment:
The style here looks a bit off tbh

Reply:
> So a few things before its good to merge

i will do those changes later at 9 pm, need to prepare for an interview.